releases/v0.4.0.md

```diff
@@ -0,0 +1,67 @@
+# Release Notes for v0.4.0
+Release Date: 2023-03-03 
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
+
+## What's new
+- Skopio and umoci dependencies are removed with our image-rs component fully integrated
+- //add more new stuff here
+
+
+## Hardware Support
+Confidential Containers is tested with attestation on the following platforms:
+- Intel TDX
+- AMD SEV
+
+The following platforms are untested or partially supported:
+- Intel SGX
+- AMD SEV-ES
+- IBM Secure Execution (SE) on IBM zSystems & LinuxONE
+
+The following platforms are in development:
+- AMD SEV-SNP
+
+## Limitations
+
+The following are known limitations of this release:
+
+- Platform support is currently limited, and rapidly changing
+  * AMD SEV-ES is not tested in the CI.
+  * Image signature validation has not been tested with AMD SEV.
+  * s390x does not support cosign signature validation
+- SELinux is not supported on the host and must be set to permissive if in use.
+- Attestation and key brokering support is still under development
+  * The disk-based key broker client (KBC) is used for non-tee testing, but is not suitable for production, except with encrypted VM images.
+  * Currently, there are two KBS that can be used:
+    - simple-kbs:  simple key broker service (KBS) for SEV(-ES).
+    - [Verdictd](https://github.com/inclavare-containers/verdictd): An external project with which Attestation Agent can conduct remote attestation communication and key acquisition via EAA KBC
+  * The full-featured generic KBS and the corresponding KBC are still in the development stage.
+  * For developers, other KBCs can be experimented with.
```

Comment:
Does this refer to our disk KBC, or do we have other examples we can share here?